docs/pages/base/react-badge/[docsTab]/index.js

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import MarkdownDocs from 'docs/src/modules/components/MarkdownDocsV2';
+import AppFrame from 'docs/src/modules/components/AppFrame';
+import * as pageProps from 'docs/data/base/components/badge/badge.md?@mui/markdown';
+import mapApiPageTranslations from 'docs/src/modules/utils/mapApiPageTranslations';
+import BadgeUnstyledApiJsonPageContent from '../../api/badge-unstyled.json';
+import useBadgeApiJsonPageContent from '../../api/use-badge.json';
+
+export default function Page(props) {
+  const { userLanguage, ...other } = props;
+  return <MarkdownDocs {...pageProps} {...other} />;
+}
+
+Page.getLayout = (page) => {
+  return <AppFrame>{page}</AppFrame>;
+};
+
+export const getStaticPaths = () => {
+  return {
+    paths: [{ params: { docsTab: 'components-api' } }, { params: { docsTab: 'hooks-api' } }],
+    fallback: false, // can also be true or 'blocking'
+  };
+};
+
+export const getStaticProps = () => {
```

Comment:
In the past, we struggled to make the translation work with this approach: https://github.com/vercel/next.js/issues/15315#issuecomment-735275969. I'm curious to see which translation approach React takes in https://github.com/reactjs/react.dev/issues/4135.